### ISSUE TITLE
Game Crash when loading a village

### ISSUE THREAD

**Issue by NisltheKid** [EnderMediaDev/The-1.7.10-Pack#927]
Hi,

I'm playing the modpack and it is very cool. I had a lot off fun with it until now:

Some chunks south-east from my base is a village. I've visited this village some time ago and everything was ok, but now everytime when I come Close enough to the viallage so the chunk is loaded the game crashes. When I then reload this world it chrashes directly (when the chunk with the viallge is loaded I think). I have to decrease the number of loaded chunks, then I can load the world and fly away, but this only bypasses the problem and didn't fix it.

Here is the crash report:
**EDITED: Please use a paste-tool: http://pastebin.com/ngsSZRWi**
I hope there is a way to fix this problem.

With kind regards, Nisl the Kid

**Comment by xJon** [comment]
What version of the modpack are you playing with? Did you modify it?

**Comment by NisltheKid** [comment]
I'm playing version 0.8.1 without any modifications. Just downloaded and installed it with the Technic Launcher.

**Comment by xJon** [comment]
Very weird. Try to report the issue with the crash report [there](https://github.com/MrTJP/MrTJPCore/issues), as it somehow related to the log. Not more than I can recommend.

**Comment by strmrts** [comment]
Just wanted to mention I had a village problem that sounds similar today. I settled close to a village which didn't have any problems, but now much later I've been getting extreme lag (~7 fps) whenever I entered a certain number of chunks. Turns out several dozen villagers had all been spawning in one house. Destroying it fixed the issue.

Sorry to hear you are crashing. You might be able to use MCedit with a bit of fiddling
